backend/src/mqtt/hardware/events/sensorEventMapper.ts

```diff
@@ -79,24 +79,25 @@ export class SensorEventMapper implements EventMapper<SensorEventType> {
 
   private _updateLeds(teamColor: TeamColor) {
     const dkMqttClient: DkMqttClient = DkMqttClient.getInstance();
+
+    let ledColors = new Array<string>()
+
+    for (let i = 0; i < this._tableHandler.subject.game.getTeamByColor(teamColor).score; i++) {
+      if (i % 2 == 0) {
+        ledColors.push("0x236932", "0x236932")
```

Comment:
Define the colors used here as named constants

Reply:
Maybe we could also make that configurable which colors are shown with the env?

Reply:
Making the colors configurable should be an extra issue because I changed the .env file to a .yaml in this [pull request](https://github.com/DigKick/DigKick/pull/45)